### ISSUE TITLE
[subviews] movement destination offset

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#976]
Looks like a recent subviews regression (since the last rotation change?). When clicking to move, the destination is always offset from the cursor position, even when there's no blocking stuff around.

Eg. click and what the destination is set to:
![image](https://user-images.githubusercontent.com/121515/95648859-b2921b00-0ada-11eb-8036-c41745571ed4.png)

@bernds

**Comment by bernds** [comment]
I think I saw that once while working on a different patch, but couldn't reproduce it. Does this happen with left clicks, right clicks or both?

**Comment by lynxlynxlynx** [comment]
Either click works, but rightclick is additionally interesting — if you don't move the mouse and continue clicking, the subsequent reticles will circle around the cursor.

**Comment by bernds** [comment]
Is this when moving single characters, and only with certain formations? Does the following fix it for you?

```diff
diff --git a/gemrb/core/GUI/GameControl.cpp b/gemrb/core/GUI/GameControl.cpp
index bb4b4a589..67f2c67e3 100644
--- a/gemrb/core/GUI/GameControl.cpp
+++ b/gemrb/core/GUI/GameControl.cpp
@@ -2293,7 +2293,7 @@ void GameControl::CommandSelectedMovement(const Point& p, unsigned short Mod)
 			actor->Stop();
 		}
 		
-		if (party.size() > 1 || isFormationRotation) {
+		if (party.size() > 1) {
 			CreateMovement(actor, formationPoints[i], Mod & GEM_MOD_SHIFT);
 		} else {
 			CreateMovement(actor, p, Mod & GEM_MOD_SHIFT);
```

**Comment by lynxlynxlynx** [comment]
Single actor, of course. And yes, that fixes it. I'll push it as you to save the need for a PR.

**Comment by bernds** [comment]
Ok. What happened was that I tested all the formations, but only with group movement. For single-character movement I must have always had my default formation on where it doesn't make a difference.
There's one more oddity I just spotted, which seems to have been in the original code already. In OnMouseUp, CommandSelectedMovement gets called with two different points depending right-click or left-click. It gets initialized as
`        Point p = ConvertPointFromScreen(me.Pos()) + vpOrigin;`
but the right-click case overrides it before dropping into CommandSelectedMovement:
`        p = gameClickPoint;`
